### DISCUSSION TITLE
Welcome to kiwaku Discussions!

### DISCUSSION THREAD

**Discussion by jfalcou** [jfalcou/kiwaku#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ThomasRetornaz** [comment]
Hi i'm a software engineer in the fields of image processing and machine learning

I plane to try/test/integrate kiwaku + eve as thrid party module of a image processing library (first :))
i've implemented a tensor like container in a rough way and i'm very interested in what kiwaku could offer
I just start to dig in code of the libraries

Somethings that coiul be very helpfull are
1. helpers/iterator/range (whatever) to iterate on the edges of a container. Think about CV algorithm which need to threat border as a specal case
2. extended view (aka something like [array_view](https://github.com/wardw/array_view) which i have integrated) with stride support think again CV (processing sub image, volume, crop) and ML (eg Covolutionnal Neural Nets with strided, diluted convolutions)

I have some spart time so i could also contribute on MSVC support and tests as a start

Regards
TR

**Reply by jfalcou** [reply]
Hi Thomas,
we will be putting up a road map issue soon. 
Most of your points are beign worked on atm (the slice view beign almost complete) so I'll gladly hear about whatever use cases you may want Kiwaku to support.

With respect to Kiwaku/EVE interaction, on epiece of Kiwaku will be to provide customisable algorithm able to consume context, ie hitns on how to apply the operations. THis should be enough to support algorithmic parallelization.
